src/components/DetailsPanelFunction.vue

```diff
@@ -11,7 +11,11 @@
         </v-details-button>
       </template>
     </v-details-panel-header>
-    <v-details-panel-list title="Events" :items="object.events" />
+    <v-details-panel-list
+      title="Events"
+      :items="object.events"
+      :event-quickview="true"
```

Comment:
This property should be present for any detail panel that lists events (routes, labels, etc.). To that point, should the quick view icon just be present for any list item that contains an event? It feels like we're missing some abstraction here.

Reply:
Done: https://github.com/applandinc/appmap-js/pull/168/commits/cef7b01d1182af41ae67c17bbf5accc41053f1df